### PR TITLE
create a new file and write a function that describe how a junior and senior write code

### PR DESCRIPTION
**Pull request by rohan-sorkar**

### CODE REVIEW

File: hello.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// junior vs pro
```
Comment (mohammadsaju) [inline hello.js:1]: can you replace ''pro" into "professional"

Comment (rohan-sorkar) [reply hello.js:1]: yeah sure ☺

**Review by mohammadsaju — CHANGES_REQUESTED** [review]
please change the code as quick as possible

**Review by mohammadsaju — APPROVED** [review]
well done my boy

**Pull request merged**